### PR TITLE
#24 Put browser clients on the same libp2p network as Storage Miners

### PR DESCRIPTION
**Pull request by aistoc**

### CODE REVIEW

**Review by ionutiorgulescu — APPROVED** [review]

File: src/background/Node.js [hunk 1]
```diff
@@ -29,9 +30,10 @@ class Node {
   queriedCids = new Set();
 
   async initialize({ rendezvousIp, rendezvousPort, wallet }) {
-    const rendezvousProtocol = /^\d+\.\d+\.\d+\.\d+$/.test(rendezvousIp) ? 'ip4' : 'dns4';
-    const rendezvousWsProtocol = `${rendezvousPort}` === '443' ? 'wss' : 'ws';
-    const rendezvousAddress = `/${rendezvousProtocol}/${rendezvousIp}/tcp/${rendezvousPort}/${rendezvousWsProtocol}/p2p-webrtc-star`;
+    // ToDo: remove these lines at cleanup
+    // const rendezvousProtocol = /^\d+\.\d+\.\d+\.\d+$/.test(rendezvousIp) ? 'ip4' : 'dns4';
+    // const rendezvousWsProtocol = `${rendezvousPort}` === '443' ? 'wss' : 'ws';
+    // const rendezvousAddress = `/${rendezvousProtocol}/${rendezvousIp}/tcp/${rendezvousPort}/${rendezvousWsProtocol}/p2p-webrtc-star`;
```
Comment (brunolm) [inline src/background/Node.js:37]:
```suggestion
```

**Review by brunolm — APPROVED** [review]

**Comment by brunolm** [comment]
Not sure if in scope here, but if the `rendezvousAddress` is not going to be used anymore there should be an issue to remove it from the options I think.

**Comment by ghost** [comment]
Thanks for thinking of that @brunolm.  I've made a new issue #35 for this.

**Review by ghost — APPROVED** [review]

File: src/background/Node.js (hunk 1 above)
Comment (ghost) [reply src/background/Node.js:37]: @brunolm Apologies, I merged without this change.  I moved this cleanup into issue #35.

**Pull request merged**